ceph.spec.in

```diff
@@ -403,7 +402,6 @@ Summary:	OCF-compliant resource agents for Ceph daemons
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
-License:	LGPL-2.0
```

Comment:
The key words here are "may differ . . . if necessary"

If we are going to declare a separate/difference license for a subpackage, there would need to be some justification why it is "necessary".